src/applications/my-education-benefits/config/chapters/33/contactInfo/mailingAddress.js

```diff
@@ -125,6 +125,20 @@ const mailingAddress33 = {
             (errors, field) => {
               if (isOnlyWhitespace(field)) {
                 errors.addError('Please enter your full street address');
+              } else if (field?.length < 3) {
```

Comment:
Extraneous change to another application

Reply:
so weird, i did a rebase. will revert